bylaws/003-membership.md

```diff
@@ -98,9 +98,13 @@ will be assumed.
 
 If, in the judgement of PHP-FIG, a Voting Representative is acting
 inappropriately and to the detriment of PHP-FIG's ability to meet its
-objectives, a vote may be taken to request a replacement Voting Representative in
-accordance with the [Voting Protocol bylaw][voting] or to expel the Member
-Project where replacing a Voting Representative is not possible.
+objectives, a vote may be taken to request a replacement Voting Representative,
+or to expel the Member Project where replacing a Voting Representative is not
+possible.
+
+The expulsion of a Voting Representative requires a vote in accordance with the
```

Comment:
It certainly looks that way, but if you read the document in entirety you'll notice two sections, one on expelling voting members and one on member projects. I used the same paragraph for each, because the same is true of each. I didn't want to worry about DRY or abstraction :D